python/glchat-sdk/glchat_sdk/conversation.py

```diff
@@ -0,0 +1,133 @@
+"""Conversation handling for the GLChat Python client.
+
+This module provides the ConversationAPI class for handling conversation operations
+with the GLChat backend, including creating new conversations.
+
+Authors:
+    Hermes Vincentius Gani (hermes.v.gani@gdplabs.id)
+
+References:
+    None
+"""
+
+import logging
+from typing import Any
+from urllib.parse import urljoin
+
+import httpx
+
+from glchat_sdk.models import ConversationRequest
+
+logger = logging.getLogger(__name__)
+
+
+class ConversationAPI:
+    """Handles conversation API operations for the GLChat API."""
+
+    def __init__(self, client):
+        self._client = client
+
+    def _validate_inputs(self, user_id: str, chatbot_id: str) -> None:
+        """Validate input parameters.
+
+        Args:
+            user_id (str): User identifier
+            chatbot_id (str): Chatbot identifier
+
+        Raises:
+            ValueError: If user_id or chatbot_id is empty
+        """
+        if not user_id:
+            raise ValueError("user_id cannot be empty")
+        if not chatbot_id:
+            raise ValueError("chatbot_id cannot be empty")
+
+    def _prepare_request_data(
+        self,
+        user_id: str,
+        chatbot_id: str,
+        title: str | None = None,
+        model_name: str | None = None,
+    ) -> dict[str, Any]:
+        """Prepare request data for the API call.
+
+        Args:
+            user_id (str): Required user identifier
+            chatbot_id (str): Required chatbot identifier
+            title (str | None): Optional conversation title
+            model_name (str | None): Optional model name to use
+
+        Returns:
+            dict containing the prepared request data
```

Comment:
```suggestion
            dict[str, Any]: Dictionary containing the prepared request data
```
and update the other function docstrings on this file